inception-project/inception
# #988 Make max. results configurable for ElasticSearch repos

**What's in the PR**
* Max. number of returned results for external search is now configurable in the repository traits. The hard-coded upper limit is 25000.
* Min. number is also hard-coded at 1.
* resolves #988 

**How to test manually**
* Enter "Document Repositories" tab in project settings
* Specify result size (play around and try illegal numbers)
* Execute external search query and observe whether number of returned results matches the specified result size

**Automatic testing**

**Documentation**
